pygmt/helpers/testing.py

```diff
@@ -157,6 +157,8 @@ def download_test_data():
         "@earth_relief_10m_p",
         "@earth_relief_05m_p",
         "@earth_relief_05m_g",
+        "@earth_relief_01m_p",
+        "@earth_relief_01m_g",
```

Comment:
`earth_relief_01` is about 258MB. See https://docs.generic-mapping-tools.org/latest/datasets/remote-data.html#global-earth-relief-grids. Do we want to cache the whole dataset?

Reply:
I changed it to the 3 arc min grid. Hope that's fine?